File: Igtampe.CDBFS.Common/CdbfsFile.cs
namespace Igtampe.CDBFS.Common {
    public class CdbfsFile : Editable {

        public int Id { get; set; } = 0;
        public int Drive { get; set; } = 0;
        public int? Folder { get; set; } = 0;
        public string Name { get; set; } = "";
        public int Size { get; set; } = 0;
        public string MimeType { get; set; } = "";
    }
}

File: igtampe.cdbfs.frontend/src/model/CdbfsFile.ts
import Editable from "./Editable";

export default class CdbfsFile extends Editable {
    public constructor(
        public id: number,
        public drive: number,
        public folder: number,
        public name: string,
        public size: number,
        public mimeType: string,
        createTs: string, createUserId: string, updateTs: string, updateUserId: string,
    ) { super(createTs, createUserId, updateTs, updateUserId) }
}